TelCo.ColorCoder/testmanual.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+namespace TelCo.ColorCoder{
+
+class testmanual:IReferenceManual
+    {
+        private int max_length = colorFromPairnum.colorMapMajor.Length;
+        private int min_length = colorFromPairnum.colorMapMinor.Length;
+        public  void manual()
+        {            Console.WriteLine("Testing reference manual/n **********/n");
+             int pno = 1;
+            for (int i = 0; i < max_length; i++)
+            {
+                for (int j = 0; j < min_length; j++)
+                {
+                    ColorPair testPair = colorFromPairnum.GetColorFromPairNumber(pno);
+                    Debug.Assert(testPair.majorColor == colorFromPairnum.colorMapMajor[i]);
```

Comment:
The production code and test code are interleaved here.
Next time, separate the production and test code into different files. We don't want test code to be delivered and run on the customer end.